app/views/root/_base_page.html.erb

```diff
@@ -21,5 +21,7 @@
 </main>
 
 <% if local_assigns.fetch(:show_related_items, true) %>
-  <div id="related-items"></div>
+  <aside>
+    <div id="related-items"></div>
```

Comment:
Because of slimmer - https://github.com/alphagov/slimmer/blob/master/lib/slimmer/processors/related_items_inserter.rb.  It specifically looks for a `div#related-items`, although the selector there could probably be relaxed to just `#related-items`.

Also, from looking at that slimmer processor, I don't think this will do what you intend.  It just uses the presence of this div as a flag to trigger the insertion of related items into the static template.